services/monitor/monitorChains.json

```diff
@@ -194,5 +194,16 @@
       "https://rpc.ankr.com/scroll_sepolia_testnet",
       "https://scroll-sepolia-rpc.publicnode.com"
     ]
+  },
+  {
+    "name": "Zircuit Mainnet",
+    "chainId": 48900,
+    "rpc": [
+      {
+        "type": "ApiKey",
+        "url": "https://lb.drpc.org/ogrpc?network=zircuit-mainnet&dkey={API_KEY}",
+        "apiKeyEnvName": "ZIRCUIT_API_KEY"
+      }
+    ]
```

Comment:
Why aren't you using public RPCs like others? Does Zircuit on drpc specifically require an API key?

Reply:
Zircuit gave us a private RPC